docs/release/releases/release-1.10.md

```diff
@@ -0,0 +1,36 @@
+# Cluster API v1.10
+
+## Timeline
+
+The following table shows the preliminary dates for the `v1.10` release cycle.
+
+| **What**                                             | **Who**      | **When**                    | **Week** |
+|------------------------------------------------------|--------------|-----------------------------|----------|
+| Start of Release Cycle                               | Release Lead | Monday 16th December 2024   | week 1   |
+| Schedule finalized                                   | Release Lead | Friday 20th December 2024   | week 1   |
+| Team finalized                                       | Release Lead | Friday 20th December 2024   | week 1   |
+| *v1.8.x & v1.9.x released*                           | Release Lead | Tuesday 21th January 2025   | week 6   |
+| *v1.8.x & v1.9.x released*                           | Release Lead | Tuesday 18th February 2025  | week 10  |
+| v1.10.0-beta.0 released                              | Release Lead | Tuesday 18th March 2025     | week 14  |
+| Communicate beta to providers                        | Comms Manager| Tuesday 18th March 2025     | week 14  |
+| *v1.8.x & v1.9.x released*                           | Release Lead | Tuesday 18th March 2025     | week 14  |
+| v1.10.0-beta.x released                              | Release Lead | Tuesday 25th March 2025     | week 15  |
+| KubeCon idle week                                    |      -       | 1st - 4th April 2025        | week 16  |
+| release-1.10 branch created (**Begin [Code Freeze]**)| Release Lead | Tuesday 8nd April 2025      | week 17  |
+| v1.10.0-rc.0 released                                | Release Lead | Tuesday 8nd April 2025      | week 17  |
+| release-1.10 jobs created                            | CI Manager   | Tuesday 8nd April 2025      | week 17  |
+| v1.10.0-rc.x released                                | Release Lead | Tuesday 15th April 2025     | week 18  |
+| **v1.10.0 released**                                 | Release Lead | Tuesday 22th April 2025     | week 19  |
+| *v1.8.x & v1.9.x released*                           | Release Lead | Tuesday 22th April 2025     | week 19  |
+| Organize release retrospective                       | Release Lead | TBC                         | week 19  |
```

Comment:
`| *v1.10.1 released (tentative)*                         | Release Lead | Tuesday 29th April 2025  | week 20  |`

Let's add a placeholder line for surfacing that we plan to cut a a patch as soon as we can test with K8s v1.33.0

Reply:
done